cob_hardware_config/robots/cob4-9/config/scan_unifier_config.yaml

```diff
@@ -0,0 +1,4 @@
+#Scan_unifier_config-parameters
+
+input_scans: ["base_laser_front/scan","base_laser_left/scan","base_laser_right/scan"]
+loop_rate: 100.0
```

Comment:
`loop_rate` is no longer a parameter needed by the `scan_unifier` (https://github.com/ipa320/cob_driver/pull/324/files#diff-4b8c9c7e0bb817a56030c63677f60462L96)
we should remove it for all robots